script/upgrades/MU03/MU03Checks.sol

```diff
@@ -362,31 +360,27 @@ contract MU03Checks is Script, Test {
     console2.log("\tBreakers set with trading mode 3");
 
     // verify that rate feed dependencies were configured correctly
-    address cEurDependency = BreakerBox(breakerBox).rateFeedDependencies(cEURUSDCConfig.referenceRateFeedID, 0);
-    address cBrlDependency = BreakerBox(breakerBox).rateFeedDependencies(cBRLUSDCConfig.referenceRateFeedID, 0);
-    require(cEurDependency == cUSDUSDCConfig.referenceRateFeedID, "cEUR/USDC dependency not set correctly");
-    require(cBrlDependency == cUSDUSDCConfig.referenceRateFeedID, "cBRL/USDC dependency not set correctly");
+    address cEurDependency = BreakerBox(breakerBox).rateFeedDependencies(config.cEURUSDC.referenceRateFeedID, 0);
```

Comment:
Because this is more RateFeed related and not pool related I would use those configs instead, i.e.:
```solidity
address USDCEURDependency0 = BreakerBox(breakerBox).rateFeedDependencies(config.USDCEUR.rateFeedID, 0);
require(USDCEURDependency0, config.USDCUSD.rateFeedID);
```

Also naming it cEUR dependency I find potentially confusing. What the system is actually saying is that the USDCEUR rate feed is dependant on the USDCUSD here, the pools themselves, and thus cEUR doesn't really factor in necessarily.